app/build.gradle

```diff
@@ -19,12 +19,18 @@ android {
     }
 }
 
+final ANDROID_SUPPORT_VERSION = '25.1.0'
+
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
+
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    compile 'com.android.support:appcompat-v7:25.1.0'
-    compile 'com.android.support:support-v4:25.1.0'
+    annotationProcessor 'com.jakewharton:butterknife-compiler:8.5.1'
```

Comment:
But he is using android-plugin version of apt, which is the "native" one, which is cool. We don't do this because (I think) Jack is incompatible with Lombok.